binplace.targets

```diff
@@ -88,17 +87,35 @@
     </Copy>
   </Target>
 
-  <Target Name="BinPlaceRuntimeAssembly" DependsOnTargets="GetBinplaceDirs;GetBinplaceItems">
-    <Copy Condition="'@(RuntimeDir)' != ''"
-          SourceFiles="@(BinplaceItem)"
-          DestinationFolder="%(RuntimeDir.Identity)"
-          SkipUnchangedFiles="true"
-          OverwriteReadOnlyFiles="true"
-          Retries="$(CopyRetryCount)"
-          RetryDelayMilliseconds="$(CopyRetryDelayMilliseconds)"
-          UseHardlinksIfPossible="$(BinplaceUseHardlinksIfPossible)">
-      <Output TaskParameter="DestinationFiles" ItemName="FileWrites" />
-    </Copy>
+  <UsingTask TaskName="SaveItems" AssemblyFile="$(CoreFxToolsTaskDir)CoreFx.Tools.dll"/>
+  <Target Name="BinplaceProps"
```

Comment:
Yeah.  We can do so when merging binplace.targets to buildtools.  I didn't want to fork packaging in corefx yet since I've needed to make cross-repo changes to both packaging targets and tasks recently.